tests/api_tests.py

```diff
@@ -29,42 +29,28 @@ def jwtHeader(self):
 
     def test_default_legendgraphic(self):
         params = {
-            'service': 'WMS',
-            'version': '1.3.0',
-            'request': 'GetLegendGraphic',
             'format': 'image/png',
-            'layer': 'test_poly',
-            'crs': 'EPSG%3A2056',
-            'dpi': '96',
-            'width': '64',
-            'height': '64',
-            'bbox': '2606082.333333333%2C1233466.3333333333%2C2633175.666666666%2C1245234.9999999998',
+            'layer': 'edit_polygons'
         }
-        response = self.app.get('/somap?' + urlencode(params), headers=self.jwtHeader())
+        response = self.app.get('/qwc_demo?' + urlencode(params), headers=self.jwtHeader())
         self.assertEqual(200, response.status_code, "Status code is not OK")
```

Comment:
Does this thest make sense, @mwa? According to the README, a `GetLegendGraphic` request should be made.

Reply:
The Legend service may be called with a regular WMS `GetLegendGraphic` request, but the `SERVICE`, `REQUEST` and `VERSION` params are not in the API and thus ignored.

E.g. both these requests return the same result:
* `http://qwc-legend-service/qwc_demo?SERVICE=WMS&REQUEST=GetLegendGraphic&LAYER=qwc_demo&FORMAT=image/png`
* `http://qwc-legend-service/qwc_demo?LAYER=qwc_demo&FORMAT=image/png`

Reply:
@mwa I added `SERVICE` and the `REQUEST`  parameter for both tests.